### PR TITLE
add Daichi Furiya

### PR DESCRIPTION
**Pull request by wasabeef**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -77,7 +77,7 @@
 - [Chris Tankersley](https://github.com/dragonmantank/ama) - PHP developer, system admin, speaker, teacher, sometimes writer, and I play video games on YouTube.
 - [Deepak Verma](https://github.com/deepaknverma/ama) - Full Stack developer at [Hometrack Australia](http://hometrack.com), AWS Solution Architect, and passionate about [photography](https://www.flickr.com/photos/deepak-verma).
 - [Dan Abramov](https://github.com/gaearon/ama) - JavaScript and React hobbyist, creator of [React Hot Loader](https://github.com/gaearon/react-hot-loader) and [Redux](https://github.com/rackt/redux).
-
+- [Daichi Furiya (Wasabeef)](https://github.com/wasabeef/ama) - Software Developer in Japan. I love Android and JavaScript, iOS.
```
Comment (shinnn) [inline readme.md]: `Android, JavaScript and iOS`

**Comment by wasabeef** [comment]
@shinnn 

Thank you.

**Comment by shinnn** [comment]
https://github.com/sindresorhus/amas/blob/master/create-ama.md

> If you forked, you have to enable the issue tracker: `Settings` → `Features` → `☑ Issues`.

You haven't yet.

**Comment by wasabeef** [comment]
@shinnn 
Done :+1:

File: readme.md [hunk 2]
```diff
@@ -77,7 +77,7 @@
 - [Chris Tankersley](https://github.com/dragonmantank/ama) - PHP developer, system admin, speaker, teacher, sometimes writer, and I play video games on YouTube.
 - [Deepak Verma](https://github.com/deepaknverma/ama) - Full Stack developer at [Hometrack Australia](http://hometrack.com), AWS Solution Architect, and passionate about [photography](https://www.flickr.com/photos/deepak-verma).
 - [Dan Abramov](https://github.com/gaearon/ama) - JavaScript and React hobbyist, creator of [React Hot Loader](https://github.com/gaearon/react-hot-loader) and [Redux](https://github.com/rackt/redux).
-
+- [Daichi Furiya (Wasabeef)](https://github.com/wasabeef/ama) - Software Developer in Japan. I love Android, JavaScript and iOS.
```
Comment (shinnn) [inline readme.md]: > `Daichi Furiya (Wasabeef)`

@sindresorhus Adding nickname like this format is allowed?

**Comment by sindresorhus** [comment]
Only full name please.

**Comment by wasabeef** [comment]
@sindresorhus 
Is it acceptable for you?

**Comment by sindresorhus** [comment]
I would prefer without the nickname sentence. You can have that in your AMA readme instead.

**Comment by wasabeef** [comment]
@sindresorhus 
Thanks for everything.

**Comment by sindresorhus** [comment]
Hey. Want to ask you a favor. Would be cool if you could ask some questions on the other AMAs in the list (not my AMA). I think AMAs are a great opportunity to learn from each other. Asking a question is also a good way to encourage receiving one ;)

For inspiration of what to ask, you could take a look at the questions in some other AMAs, like:
https://github.com/passy/ama/issues?q=
https://github.com/sebmck/ama/issues?q=
https://github.com/wesbos/ama/issues?q=
https://github.com/sindresorhus/ama/issues?q=

**Pull request closed without merging**